app/vmalert/tpl/footer.qtpl

```diff
@@ -37,6 +37,15 @@
               if (group.length > 0) {
                 group.click();
               }
+
+
+            });
+
+            $(document).ready(function() {
+              var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
+              var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
+                return new bootstrap.Tooltip(tooltipTriggerEl)
+              })
```

Comment:
I think this  part can be simplified to
```javascript
$('[data-bs-toggle="tooltip"]').tooltip();
```
It will select all and initialize all tooltips on a page